docs/source/install-tutorial.rst

```diff
@@ -58,6 +83,7 @@ add this content to ``/home/bepasty/bin/gunicorn_bepasty``:
   SOCKFILE=$HOME/gunicorn.sock  # we will communicate using this unix socket
   PIDFILE=$HOME/gunicorn.pid
   NUM_WORKERS=3                 # how many worker processes should Gunicorn spawn
+  export PATH="/home/bepasty/.local/bin:$PATH"  #include gunicorn path for exec
```

Comment:
always have 2 spaces (at least) before and 1 space after `#`.

Reply:
OK - understood.